src/DependencyInjection/Configuration.php

```diff
@@ -45,6 +45,9 @@ public function getConfigTreeBuilder()
                         ->scalarNode('debug_id_header_key')->defaultNull()->end()
                     ->end()
                 ->end()
+                ->arrayNode('http_listener_skipped_routes')
+                    ->prototype('scalar')->end()
+                ->end()
```

Comment:
```suggestion
                ->arrayNode('http_listener_skipped_routes')
                    ->beforeNormalization()
                        ->castToArray()
                    ->end()
                    ->prototype('scalar')->end()
                ->end()
```

after this you can write config like this:

```yml
  http_listener_skipped_routes: health
```

or like this
```yml
  http_listener_skipped_routes: 
    - health
    - some_other_route
```

And is goona be an array in both cases

Reply:
done